3001-4000/LIT3058RepCh338.xml

```diff
@@ -1,16 +1,12 @@
-<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
-schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="work" xml:lang="en" xml:id="LIT3058RepCh338">
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1">Hymn to Our Lady Mary (ʾƎsaggǝd laki)</title>
-            
-            <title xml:id="t2" xml:lang="gez" type="main">እሰግድ፡ ለኪ፡ ወእዌድሰኪ፡
-            </title>
-            <title corresp="#t2" xml:lang="gez" type="normalized">ʾƎsaggǝd laki waʾǝweddǝsaki</title>
-            <title corresp="#t2" xml:lang="en">I worship
-               Thee…</title>
+            <title xml:lang="en" xml:id="t1" type="main">I worship Thee… </title>
```

Comment:
If you want "I worship thee..." to be the printed title, I would suggest:
```
<title xml:lang="gez" xml:id="t1">እሰግድ፡ ለኪ፡ እሰግድ፡ ለኪ፡ እሰግድ፡ ለኪ፡ ወእዌድሰኪ፡ </title>
<title  xml:lang="gez" corresp="#t1" type="normalized">ʾƎsaggǝd laki ʾǝsaggǝd laki ʾǝsaggǝd laki wa-ʾǝweddǝsaki</title>
<title xml:lang="en" corresp="#t1">I worship you, I worship you, I worship you, and I praise you… </title>
<title xml:lang="en" corresp="#t1" type="main">I worship Thee… </title>
```

Reply:
I want to have it as: I worship Thee... (እሰግድ፡ ለኪ፡ እሰግድ፡ ለኪ፡ እሰግድ፡ ለኪ፡ ወእዌድሰኪ፡)
The words እሰግድ፡ ለኪ፡ are repeated three times. "I worship Thee..." is a kind of common English title.

Reply:
then my proposal should work